R/correlation.R

```diff
@@ -318,6 +325,18 @@ correlation <- function(data,
     }
   }
 
+  missing <- insight::validate_argument(missing, options = c("keep.pairwise", "keep.complete"))
```

Comment:
I'm going to be pendantic but we don't typically use dots in the easyverse. I think that you added the `keep.` in precision for a future where we might have `impute.` or something which makes sense.

Maybe `keep_pairwise` and `keep_complete`?

Reply:
I mean, this is a string - so really it can even be `c("keep pairwise", "keep complete")`.

Reply:
![](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExOWl6OGNocmw3aHlrNTFybjcyb3FuZ3JnZWNlMGl5OXdidnV5cjJlciZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/F41xyZHAZTc3bJyCJr/giphy.gif)